packages/manager/src/features/CloudPulse/shared/CloudPulseCustomSelect.tsx

```diff
@@ -227,15 +230,13 @@ export const CloudPulseCustomSelect = React.memo(
             ? ''
             : placeholder || 'Select a Value'
         }
-        textFieldProps={{
-          hideLabel: true,
-        }}
         autoHighlight
         disabled={isAutoCompleteDisabled}
         errorText={staticErrorText}
         isOptionEqualToValue={(option, value) => option.label === value.label}
-        label={placeholder || 'Select a Value'}
+        label={label || 'Select a Value'}
```

Comment:
can we put a empty value check as well
```suggestion
        label={(label & label.length > 0) ? label : 'Select a Value'}
```